packages/flutter/lib/src/cupertino/segment_control.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/painting.dart';
+
+class SegmentControlItem {
+  final String title;
+  final Widget content;
+
+  SegmentControlItem(this.title, this.content);
+}
+
+class SegmentControl extends StatefulWidget {
+  final List<SegmentControlItem> tabs;
```

Comment:
The terminology [they](https://developer.apple.com/documentation/uikit/uisegmentedcontrol) used everywhere is segments. Let's just call this segments.